test/e2e/specs/specs.go

```diff
@@ -99,6 +99,12 @@ const (
 	pollTimeout      = 1 * time.Minute
 	pollIntervalSlow = 10 * time.Second
 	pollTimeoutSlow  = 20 * time.Minute
+
+	backoffDuration = 5 * time.Second
+	backoffFactor = 2.0
+	backoffCap = 10 * time.Minute
+	backoffSteps = 8
+	backoffJitter = 0.1
```

Comment:
IIUC, cap is the max duration you wait between retries, not the total. So this would be 5 + 10 + 20 + 40 + 80 + 160 + 320 + 600 = 1235 seconds, or about 20.5 minutes.

See the documentation here: https://pkg.go.dev/k8s.io/apimachinery/pkg/util/wait#Backoff